server/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "utseus_api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "engines": {
+    "node": ">=4.0.0"
+  },
+  "dependencies": {
+    "cors": "^2.8.4",
+    "debug": "^3.1.0",
+    "ethereum-input-data-decoder": "0.0.3",
+    "ethereumjs-testrpc": "^6.0.3",
+    "promise": "^8.0.1",
+    "truffle": "^4.0.4",
+    "web3": "^0.19.1"
+  },
+  "devDependencies": {
+    "chai": "^4.1.2",
+    "eslint": "4.13.0",
+    "eslint-plugin-node": "5.2.1",
+    "express": "^4.16.2",
+    "jest": "^22.0.4",
+    "mocha": "^4.1.0",
+    "request": "^2.83.0"
+  },
+  "scripts": {
```

Comment:
also add a pre-commit block with a) linting b) running tests c) checking for extraneous dependencies.

one alternative: https://github.com/observing/pre-commit

Reply:
Done. (note: I have to remove tests from pre-commit as API test require establish environment which I can automate at this moment)